src/app/events/new/page.tsx

```diff
@@ -0,0 +1,110 @@
+'use client';
+import { ReactNode, useState, createContext, useContext, Dispatch, SetStateAction } from 'react'
+import { Confirm } from "@/components/newEvent/Confirm"
+import { Tickets } from '@/components/newEvent/Tickets';
+import { Description } from '@/components/newEvent/Description';
+import { Overview } from '@/components/newEvent/Overview';
+
+type StepType = "Overview" | "Description" | "Tickets" | "Confirm"
+interface NewEventType {
+    owner:string | undefined,
+    name:string | undefined,
+    briefDescription:string | undefined,
+    fullDescription:string | undefined,
+    startsAt: number | undefined,
+    endsAt: number | undefined,
+    type:string | undefined,
+    imageUrl:string | undefined,
+}
+
+
+interface newEventContextInterface {
+    step: StepType,
+    setStep:Dispatch<SetStateAction<StepType>> ,
+    newEvent: NewEventType,
+    setNewEvent: Dispatch<SetStateAction<NewEventType>>,
+    isContinueEnabled:boolean,
+    setIsContinueEnabled: Dispatch<SetStateAction<boolean>>,
+}
+
+export const newEventContext = createContext({} as newEventContextInterface)
+
+
+
+export default function CreateEvent() {
+
+    const [step, setStep] = useState<StepType>("Overview")
+    const [newEvent,setNewEvent] = useState({} as NewEventType)
+    const [isContinueEnabled, setIsContinueEnabled]= useState(false)
+
+    function handleContinue() {
+        console.log('Continue')
+        switch (step) {
+            case "Overview":
+                setStep("Description")
+        }
+    }
+
+    function ContinueButton() {
```

Comment:
This component could be moved out of the CreateEvent function to avoid recreating it on every render